ls1intum/Artemis
# `Development`: Match server and client run configurations

### Checklist
#### General

### Motivation and Context


I guess webpack is just used for the dev profile and is baked into the gradle build as for me this configuration failed by default?!

### Description

Building the webapp via gradle instead of running the deleted webpack gradle task and a weird make for the server.
The webpack gradle task was deleted [in this PR](https://github.com/ls1intum/Artemis/pull/4022).

### Steps for Testing
Try if it builds what you expect.
Do you expect a different build for developing on the server or client code?

### Review Progress





#### Code Review
#### Manual Tests
